IPython/utils/_tokenize_py2.py

```diff
@@ -80,9 +80,9 @@ def maybe(*choices): return group(*choices) + '?'
 # Tail end of " string.
 Double = r'[^"\\]*(?:\\.[^"\\]*)*"'
 # Tail end of ''' string.
-Single3 = r"[^'\\]*(?:(?:\\.|'(?!''))[^'\\]*)*'''"
```

Comment:
These are copies of the standard library `tokenize` module, with a couple of changes we needed. I'd like to eventually get rid of them and rely on the regular `tokenize` module, so I'm very reluctant to accept any changes in them. If you really think that tokenize is doing something wrong (which is quite possible - it is a bit buggy), please file an issue on the [Python bug tracker](http://bugs.python.org/) and link to it here.